# Patch to add visibility to build_config settings
# These need to be visible to other packages within googleurl

diff --git a/build_config/BUILD b/build_config/BUILD
--- a/build_config/BUILD
+++ b/build_config/BUILD
@@ -5,15 +5,18 @@ licenses(["notice"])
 config_setting(
     name = "windows_x86_64",
     values = {"cpu": "x64_windows"},
+    visibility = ["//visibility:public"],
 )
 
 bool_flag(
     name = "system_icu",
     build_setting_default = True,
+    visibility = ["//visibility:public"],
 )
 
 config_setting(
     name = "with_system_icu",
     flag_values = {
         ":system_icu": "True",
     },
+    visibility = ["//visibility:public"],
 )
